src/windows/gfx/d3d11-ctx.c

```diff
@@ -45,8 +58,185 @@ static void d3d11_ctx_get_size(struct d3d11_ctx *ctx, uint32_t *width, uint32_t
 	*height = rect.bottom - rect.top;
 }
 
+static void mty_validate_format_colorspace(struct d3d11_ctx *ctx, MTY_ColorFormat format, MTY_ColorSpace colorspace, DXGI_FORMAT *format_out, DXGI_COLOR_SPACE_TYPE *colorspace_out)
+{
+	DXGI_FORMAT format_new = DXGI_FORMAT_R8G8B8A8_UNORM;
+	DXGI_COLOR_SPACE_TYPE colorspace_new = DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709;
+
+	// Use the last known value if unspecified
+	if (format == MTY_COLOR_FORMAT_UNKNOWN) {
+		format_new = ctx->format;
+	}
+	if (colorspace == MTY_COLOR_SPACE_UNKNOWN) {
+		colorspace_new = ctx->colorspace;
+	}
```

Comment:
```suggestion
	if (format == MTY_COLOR_FORMAT_UNKNOWN)
		format_new = ctx->format;

	if (colorspace == MTY_COLOR_SPACE_UNKNOWN)
		colorspace_new = ctx->colorspace;
```

No need for brackets when a single if line